documentation/asciidoc/topics/proc_recovering_rolling_upgrades.adoc

```diff
@@ -0,0 +1,23 @@
+[id='recovering_rolling_upgrades_{context}']
+= Recovering from a failed Hot Rod rolling upgrade
+
+[role="_abstract"]
+You can roll back failed Hot Rod rolling upgrade to a previous version if the original cluster is still present.
```

Comment:
You can only rollback to the version that was installed prior to the upgrade being initiated, so maybe we should have:

"You can rollback a failed Hot Rod rolling upgrade to the previous version ..."